models/perm/access/repo_permission.go

```diff
@@ -253,6 +255,39 @@ func finalProcessRepoUnitPermission(user *user_model.User, perm *Permission) {
 	}
 }
 
+type ActionsTaskIDKeyType struct{}
+
+// ActionsTaskIDKey is the context key for actions task ID in modules without context service like lfs locks
+var ActionsTaskIDKey ActionsTaskIDKeyType
```

Comment:
see other comments

If we place this always into ctx.Value(ActionsTaskIDKey), we could remove the taskid parameter of this function.